nodes/number/scalar_mk4.py

```diff
@@ -273,6 +272,23 @@ def process(self):
 
             self.outputs[0].sv_set(result)
 
+    def ensure_enums_have_no_space(self, enums=None):
+        """
+        enums: a list of property names to check. like  self.current_op
+
+            self.ensure_enums_have_no_space(enums=[current_op])
+
+        https://github.com/nortikin/sverchok/pull/3483#discussion_r478389849
+        due to changes in EnumProperty definition "laws" individual enum identifiers must not
+        contain spaces. This function takes a list of enums that the node currently holds, and
+        makes sure the stored enum has no spaces.
+        """
+        for enum_property in enums:
+            current_value = getattr(self, enum_property)
+            if " " in current_value:
+                with self.sv_throttle_tree_update():
+                    setattr(self, enum_property, no_space(current_value))
```

Comment:
this should have used `self.sv_setattr_with_throttle` too.
```diff
-                with self.sv_throttle_tree_update():
-                    setattr(self, enum_property, no_space(current_value))
+                self.sv_setattr_with_throttle(enum_property, no_space(current_value))
```

Reply:
Don't understand what do you mean. The method actually is using throttling method. And I didn't add any changes to this method just moved it from base class.